src/cambridge.json

```diff
@@ -0,0 +1,21 @@
+{
+  "publisher": "Cambridge Core",
+  "publisher_parent": "Cambridge University Press",
+  "publisher_member": null,
+  "crossref_member": null,
+  "prefixes": ["10.1017"],
+  "urls": {
+    "pdf": "https://www.cambridge.org/core/services/aop-cambridge-core/content/view/%s.pdf"
+  },
+  "components": {
+    "html": null,
+    "doi": {
+      "regex": ".+"
```

Comment:
`.+` doesn't quite work. this publisher wants just the part after the prefix and slash. so this should work: `[A-Za-z0-9\\-\\.]+$` though I haven't tested it very widely, you might want to do that